openedx/core/djangoapps/profile_images/images.py

```diff
@@ -103,7 +96,7 @@ def validate_uploaded_image(uploaded_file):
     if not filetype:
         file_upload_bad_type = _(
             u'The file must be one of the following types: {valid_file_types}.'
-        ).format(valid_file_types=get_valid_file_types())
+        ).format(valid_file_types=_get_valid_file_types())
         raise ImageValidationError(file_upload_bad_type)
```

Comment:
These 3 strings (`file_upload_bad_type`, `file_upload_too_small`, and `file_upload_too_large`) are the values that will need to be flagged with `ugettext_noop`, unless my understanding of translation is inaccurate.